src/delete.c

```diff
@@ -237,7 +237,8 @@ exec_delete(int argc, char **argv)
 			goto cleanup;
 		}
 		rc = query_yesno(false,
-		            "\nProceed with deinstalling packages? ");
+		            "\nProceed? Note that if base (FreeBSD-) packages are deleted, "
```

Comment:
```suggestion
		            "\nProceed? Note that if base (FreeBSD-*) packages are deleted, "
```

Maybe better with the `*` wildcard.